pom.xml

```diff
@@ -3334,5 +3351,6 @@
     <module>exec</module>
     <module>drill-yarn</module>
     <module>distribution</module>
+    <module>drill-shaded</module>
```

Comment:
Remove. All artifacts in drill-shaded should be built once and published to maven. There is no need to build them in each drill build.

Reply:
I don't think that it is the right approach to publish artifacts to maven.
When we will make some changes in pom files, for example, we should deploy it again. And for the most of the contributors, it is unknown, how to deploy it to the maven, and I suppose most of us don't have an access to do that. So I think it should be built with drill build.

Reply:
What type of changes do you expect in pom.xml? Why would you prefer to build the same artifact over and over again even though it does not change?